web-common/src/features/entity-management/ResourceWatcher.svelte

```diff
@@ -1,13 +1,13 @@
 <script lang="ts">
   import { fileArtifacts } from "@rilldata/web-common/features/entity-management/file-artifacts";
-  import { createWatchFilesClient } from "@rilldata/web-common/features/entity-management/watch-files-client";
-  import { createWatchResourceClient } from "@rilldata/web-common/features/entity-management/watch-resources-client";
+  import { WatchFilesClient } from "@rilldata/web-common/features/entity-management/WatchFilesClient";
```

Comment:
**Issue**: Import statements for 'WatchFilesClient' and 'WatchResourcesClient' use incorrect paths or naming conventions. <br> **Fix**: Correct the import statements to match the actual file names and paths in the project structure. <br> **Code Suggestion**: 
 ```
 -  import { WatchFilesClient } from "@rilldata/web-common/features/entity-management/WatchFilesClient";
 -  import { WatchResourcesClient } from "@rilldata/web-common/features/entity-management/WatchResourcesClient";
 +  import { WatchFilesClient } from "@rilldata/web-common/features/entity-management/watch-files-client";
 +  import { WatchResourcesClient } from "@rilldata/web-common/features/entity-management/watch-resources-client";
 ```

Reply:
This seems to be incorrect, did it not catch the file name rename? The file name should match the class name with camel case.